plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/order-summary/order-summary-item.tsx

```diff
@@ -175,8 +175,6 @@ const OrderSummaryItem = ( { cartItem }: OrderSummaryProps ): JSX.Element => {
 					)
 				) }
 				<ProductMetadata
-					shortDescription={ shortDescription }
-					fullDescription={ fullDescription }
```

Comment:
Now, that we've removed `shortDescription` and `fullDescription`, we should also remove  them from here: https://github.com/woocommerce/woocommerce/blob/f1656676326a63bfa5c36c064cdd4a29a3137633/plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/order-summary/order-summary-item.tsx#L44-L45

Reply:
Good spot, I've removed them.